core/domain/platform_feature_services.py

```diff
@@ -194,8 +194,11 @@ def update_feature_flag(
         raise FeatureFlagNotFoundException(
             'Unknown feature flag: %s.' % feature_name)
 
+    # The default value of a feature flag is always False and that
+    # is why we are explicitly passing default_value as False.
     registry.Registry.update_platform_parameter(
-        feature_name, committer_id, commit_message, new_rules)
+        feature_name, committer_id, commit_message, new_rules,
+        default_value=False)
```

Comment:
Since this is not an optional parameter, you don't need `default_value=` here.

Reply:
Done